content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,73 @@
+---
+title: API for sidecar containers
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Sidecar containers are the secondary containers that runs along with the main
+application container within the same {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+These containers are used to enhance or extend the functionality of the primary container
+by providing additional services, or functionality such as logging, monitoring, security,
+or data synchronization, without directly altering the primary application code.
+
+<!-- body -->
+
+Starting with Kubernetes 1.28 in alpha, a feature gate named `SidecarContainers`
```

Comment:
This reads like "kubernetes 1.28 in alpha", ... so it is weird.

Reply:
updated.